From 86acc02dc7db4e70e72d95002ae013cb29a88de8 Mon Sep 17 00:00:00 2001
From: Gu Zitao <guzitao@wxiat.com>
Date: Tue, 7 Jun 2022 17:42:08 +0800
Subject: [PATCH] sw64: fix SPDX license identifier in uapi headers
Patch-mainline: NO, OTHERS
References: openEuler-22.03-LTS
Modified-by-SEL: No


Sunway inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I56U83

--------------------------------

It used to fail to make headers_install. This patch adds
Linux-syscall-note to SPDX license to fix this issue.

Signed-off-by: Gu Zitao <guzitao@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/include/uapi/asm/auxvec.h      | 2 +-
 arch/sw_64/include/uapi/asm/bitsperlong.h | 2 +-
 arch/sw_64/include/uapi/asm/byteorder.h   | 2 +-
 arch/sw_64/include/uapi/asm/compiler.h    | 2 +-
 arch/sw_64/include/uapi/asm/console.h     | 2 +-
 arch/sw_64/include/uapi/asm/errno.h       | 2 +-
 arch/sw_64/include/uapi/asm/fcntl.h       | 2 +-
 arch/sw_64/include/uapi/asm/fpu.h         | 2 +-
 arch/sw_64/include/uapi/asm/gentrap.h     | 2 +-
 arch/sw_64/include/uapi/asm/hmcall.h      | 2 +-
 arch/sw_64/include/uapi/asm/ioctl.h       | 2 +-
 arch/sw_64/include/uapi/asm/ioctls.h      | 2 +-
 arch/sw_64/include/uapi/asm/ipcbuf.h      | 2 +-
 arch/sw_64/include/uapi/asm/kvm.h         | 2 +-
 arch/sw_64/include/uapi/asm/kvm_para.h    | 2 +-
 arch/sw_64/include/uapi/asm/mman.h        | 2 +-
 arch/sw_64/include/uapi/asm/msgbuf.h      | 2 +-
 arch/sw_64/include/uapi/asm/param.h       | 2 +-
 arch/sw_64/include/uapi/asm/poll.h        | 2 +-
 arch/sw_64/include/uapi/asm/posix_types.h | 2 +-
 arch/sw_64/include/uapi/asm/ptrace.h      | 2 +-
 arch/sw_64/include/uapi/asm/reg.h         | 2 +-
 arch/sw_64/include/uapi/asm/regdef.h      | 2 +-
 arch/sw_64/include/uapi/asm/resource.h    | 2 +-
 arch/sw_64/include/uapi/asm/sembuf.h      | 2 +-
 arch/sw_64/include/uapi/asm/setup.h       | 2 +-
 arch/sw_64/include/uapi/asm/shmbuf.h      | 2 +-
 arch/sw_64/include/uapi/asm/sigcontext.h  | 2 +-
 arch/sw_64/include/uapi/asm/siginfo.h     | 2 +-
 arch/sw_64/include/uapi/asm/signal.h      | 2 +-
 arch/sw_64/include/uapi/asm/socket.h      | 2 +-
 arch/sw_64/include/uapi/asm/sockios.h     | 2 +-
 arch/sw_64/include/uapi/asm/stat.h        | 2 +-
 arch/sw_64/include/uapi/asm/statfs.h      | 2 +-
 arch/sw_64/include/uapi/asm/swab.h        | 2 +-
 arch/sw_64/include/uapi/asm/sysinfo.h     | 2 +-
 arch/sw_64/include/uapi/asm/termbits.h    | 2 +-
 arch/sw_64/include/uapi/asm/termios.h     | 2 +-
 arch/sw_64/include/uapi/asm/types.h       | 2 +-
 arch/sw_64/include/uapi/asm/unistd.h      | 2 +-
 40 files changed, 40 insertions(+), 40 deletions(-)

diff --git a/arch/sw_64/include/uapi/asm/auxvec.h b/arch/sw_64/include/uapi/asm/auxvec.h
index 5b554a593fc0..0d4baa48f2f3 100644
--- a/arch/sw_64/include/uapi/asm/auxvec.h
+++ b/arch/sw_64/include/uapi/asm/auxvec.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_AUXVEC_H
 #define _UAPI_ASM_SW64_AUXVEC_H
 
diff --git a/arch/sw_64/include/uapi/asm/bitsperlong.h b/arch/sw_64/include/uapi/asm/bitsperlong.h
index 5d2c677a86b8..712c823e23d8 100644
--- a/arch/sw_64/include/uapi/asm/bitsperlong.h
+++ b/arch/sw_64/include/uapi/asm/bitsperlong.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_BITSPERLONG_H
 #define _UAPI_ASM_SW64_BITSPERLONG_H
 
diff --git a/arch/sw_64/include/uapi/asm/byteorder.h b/arch/sw_64/include/uapi/asm/byteorder.h
index 1b1698df58ca..ededdd045e96 100644
--- a/arch/sw_64/include/uapi/asm/byteorder.h
+++ b/arch/sw_64/include/uapi/asm/byteorder.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_BYTEORDER_H
 #define _UAPI_ASM_SW64_BYTEORDER_H
 
diff --git a/arch/sw_64/include/uapi/asm/compiler.h b/arch/sw_64/include/uapi/asm/compiler.h
index e5cf0fb170fa..64786df0f266 100644
--- a/arch/sw_64/include/uapi/asm/compiler.h
+++ b/arch/sw_64/include/uapi/asm/compiler.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_COMPILER_H
 #define _UAPI_ASM_SW64_COMPILER_H
 
diff --git a/arch/sw_64/include/uapi/asm/console.h b/arch/sw_64/include/uapi/asm/console.h
index 91246b759ecf..a40cd7aeb31f 100644
--- a/arch/sw_64/include/uapi/asm/console.h
+++ b/arch/sw_64/include/uapi/asm/console.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_CONSOLE_H
 #define _UAPI_ASM_SW64_CONSOLE_H
 
diff --git a/arch/sw_64/include/uapi/asm/errno.h b/arch/sw_64/include/uapi/asm/errno.h
index 04b07f30c787..0d8438f6bd40 100644
--- a/arch/sw_64/include/uapi/asm/errno.h
+++ b/arch/sw_64/include/uapi/asm/errno.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_ERRNO_H
 #define _UAPI_ASM_SW64_ERRNO_H
 
diff --git a/arch/sw_64/include/uapi/asm/fcntl.h b/arch/sw_64/include/uapi/asm/fcntl.h
index 29c3aece8b55..99e1a31c5e86 100644
--- a/arch/sw_64/include/uapi/asm/fcntl.h
+++ b/arch/sw_64/include/uapi/asm/fcntl.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_FCNTL_H
 #define _UAPI_ASM_SW64_FCNTL_H
 
diff --git a/arch/sw_64/include/uapi/asm/fpu.h b/arch/sw_64/include/uapi/asm/fpu.h
index 9b25f97e6a3a..035ca65b1ba3 100644
--- a/arch/sw_64/include/uapi/asm/fpu.h
+++ b/arch/sw_64/include/uapi/asm/fpu.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_FPU_H
 #define _UAPI_ASM_SW64_FPU_H
 
diff --git a/arch/sw_64/include/uapi/asm/gentrap.h b/arch/sw_64/include/uapi/asm/gentrap.h
index 4345058291fb..3786b8b52add 100644
--- a/arch/sw_64/include/uapi/asm/gentrap.h
+++ b/arch/sw_64/include/uapi/asm/gentrap.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_GENTRAP_H
 #define _UAPI_ASM_SW64_GENTRAP_H
 
diff --git a/arch/sw_64/include/uapi/asm/hmcall.h b/arch/sw_64/include/uapi/asm/hmcall.h
index 524101102fb8..f10378ba99c8 100644
--- a/arch/sw_64/include/uapi/asm/hmcall.h
+++ b/arch/sw_64/include/uapi/asm/hmcall.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_HMCALL_H
 #define _UAPI_ASM_SW64_HMCALL_H
 
diff --git a/arch/sw_64/include/uapi/asm/ioctl.h b/arch/sw_64/include/uapi/asm/ioctl.h
index d62f10a6fa64..fb5267b034fc 100644
--- a/arch/sw_64/include/uapi/asm/ioctl.h
+++ b/arch/sw_64/include/uapi/asm/ioctl.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_IOCTL_H
 #define _UAPI_ASM_SW64_IOCTL_H
 
diff --git a/arch/sw_64/include/uapi/asm/ioctls.h b/arch/sw_64/include/uapi/asm/ioctls.h
index b4b23b5d14e8..db8e456290e6 100644
--- a/arch/sw_64/include/uapi/asm/ioctls.h
+++ b/arch/sw_64/include/uapi/asm/ioctls.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_IOCTLS_H
 #define _UAPI_ASM_SW64_IOCTLS_H
 
diff --git a/arch/sw_64/include/uapi/asm/ipcbuf.h b/arch/sw_64/include/uapi/asm/ipcbuf.h
index f063105ba09f..553cdb37052d 100644
--- a/arch/sw_64/include/uapi/asm/ipcbuf.h
+++ b/arch/sw_64/include/uapi/asm/ipcbuf.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_IPCBUF_H
 #define _UAPI_ASM_SW64_IPCBUF_H
 
diff --git a/arch/sw_64/include/uapi/asm/kvm.h b/arch/sw_64/include/uapi/asm/kvm.h
index 47877b56e980..54683c2bff49 100644
--- a/arch/sw_64/include/uapi/asm/kvm.h
+++ b/arch/sw_64/include/uapi/asm/kvm.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_KVM_H
 #define _UAPI_ASM_SW64_KVM_H
 
diff --git a/arch/sw_64/include/uapi/asm/kvm_para.h b/arch/sw_64/include/uapi/asm/kvm_para.h
index 405840b0e1d8..3c0f9fa712ab 100644
--- a/arch/sw_64/include/uapi/asm/kvm_para.h
+++ b/arch/sw_64/include/uapi/asm/kvm_para.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_KVM_PARA_H
 #define _UAPI_ASM_SW64_KVM_PARA_H
 
diff --git a/arch/sw_64/include/uapi/asm/mman.h b/arch/sw_64/include/uapi/asm/mman.h
index f9ac285702a5..57970e1e3a2c 100644
--- a/arch/sw_64/include/uapi/asm/mman.h
+++ b/arch/sw_64/include/uapi/asm/mman.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_MMAN_H
 #define _UAPI_ASM_SW64_MMAN_H
 
diff --git a/arch/sw_64/include/uapi/asm/msgbuf.h b/arch/sw_64/include/uapi/asm/msgbuf.h
index d61eea10813d..b938df3664a0 100644
--- a/arch/sw_64/include/uapi/asm/msgbuf.h
+++ b/arch/sw_64/include/uapi/asm/msgbuf.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_MSGBUF_H
 #define _UAPI_ASM_SW64_MSGBUF_H
 
diff --git a/arch/sw_64/include/uapi/asm/param.h b/arch/sw_64/include/uapi/asm/param.h
index 75eeac6a7dc8..16c4934c937e 100644
--- a/arch/sw_64/include/uapi/asm/param.h
+++ b/arch/sw_64/include/uapi/asm/param.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_PARAM_H
 #define _UAPI_ASM_SW64_PARAM_H
 
diff --git a/arch/sw_64/include/uapi/asm/poll.h b/arch/sw_64/include/uapi/asm/poll.h
index 5e2de3182050..114d0344e377 100644
--- a/arch/sw_64/include/uapi/asm/poll.h
+++ b/arch/sw_64/include/uapi/asm/poll.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_POLL_H
 #define _UAPI_ASM_SW64_POLL_H
 
diff --git a/arch/sw_64/include/uapi/asm/posix_types.h b/arch/sw_64/include/uapi/asm/posix_types.h
index fb7badf78c3c..182741aaa06e 100644
--- a/arch/sw_64/include/uapi/asm/posix_types.h
+++ b/arch/sw_64/include/uapi/asm/posix_types.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_POSIX_TYPES_H
 #define _UAPI_ASM_SW64_POSIX_TYPES_H
 
diff --git a/arch/sw_64/include/uapi/asm/ptrace.h b/arch/sw_64/include/uapi/asm/ptrace.h
index 7cf7bf5a75b4..96cb10891bea 100644
--- a/arch/sw_64/include/uapi/asm/ptrace.h
+++ b/arch/sw_64/include/uapi/asm/ptrace.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_PTRACE_H
 #define _UAPI_ASM_SW64_PTRACE_H
 
diff --git a/arch/sw_64/include/uapi/asm/reg.h b/arch/sw_64/include/uapi/asm/reg.h
index a19dc4cbf744..e692e45a4936 100644
--- a/arch/sw_64/include/uapi/asm/reg.h
+++ b/arch/sw_64/include/uapi/asm/reg.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_REG_H
 #define _UAPI_ASM_SW64_REG_H
 
diff --git a/arch/sw_64/include/uapi/asm/regdef.h b/arch/sw_64/include/uapi/asm/regdef.h
index 5031abc0947a..ad4475b79435 100644
--- a/arch/sw_64/include/uapi/asm/regdef.h
+++ b/arch/sw_64/include/uapi/asm/regdef.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_REGDEF_H
 #define _UAPI_ASM_SW64_REGDEF_H
 
diff --git a/arch/sw_64/include/uapi/asm/resource.h b/arch/sw_64/include/uapi/asm/resource.h
index ff7dc683c195..fecca2214849 100644
--- a/arch/sw_64/include/uapi/asm/resource.h
+++ b/arch/sw_64/include/uapi/asm/resource.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_RESOURCE_H
 #define _UAPI_ASM_SW64_RESOURCE_H
 
diff --git a/arch/sw_64/include/uapi/asm/sembuf.h b/arch/sw_64/include/uapi/asm/sembuf.h
index f574390bcd57..08b0876e739c 100644
--- a/arch/sw_64/include/uapi/asm/sembuf.h
+++ b/arch/sw_64/include/uapi/asm/sembuf.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SEMBUF_H
 #define _UAPI_ASM_SW64_SEMBUF_H
 
diff --git a/arch/sw_64/include/uapi/asm/setup.h b/arch/sw_64/include/uapi/asm/setup.h
index fefd57415a3b..10ce5dba9c30 100644
--- a/arch/sw_64/include/uapi/asm/setup.h
+++ b/arch/sw_64/include/uapi/asm/setup.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SETUP_H
 #define _UAPI_ASM_SW64_SETUP_H
 
diff --git a/arch/sw_64/include/uapi/asm/shmbuf.h b/arch/sw_64/include/uapi/asm/shmbuf.h
index 66d8cb5b2ba3..4572337bee02 100644
--- a/arch/sw_64/include/uapi/asm/shmbuf.h
+++ b/arch/sw_64/include/uapi/asm/shmbuf.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SHMBUF_H
 #define _UAPI_ASM_SW64_SHMBUF_H
 
diff --git a/arch/sw_64/include/uapi/asm/sigcontext.h b/arch/sw_64/include/uapi/asm/sigcontext.h
index c2b7cff884eb..facbf34e920d 100644
--- a/arch/sw_64/include/uapi/asm/sigcontext.h
+++ b/arch/sw_64/include/uapi/asm/sigcontext.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SIGCONTEXT_H
 #define _UAPI_ASM_SW64_SIGCONTEXT_H
 
diff --git a/arch/sw_64/include/uapi/asm/siginfo.h b/arch/sw_64/include/uapi/asm/siginfo.h
index b50afbf15f7c..4a58eea9b67c 100644
--- a/arch/sw_64/include/uapi/asm/siginfo.h
+++ b/arch/sw_64/include/uapi/asm/siginfo.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SIGINFO_H
 #define _UAPI_ASM_SW64_SIGINFO_H
 
diff --git a/arch/sw_64/include/uapi/asm/signal.h b/arch/sw_64/include/uapi/asm/signal.h
index 71471c8c7624..5bad0adae93f 100644
--- a/arch/sw_64/include/uapi/asm/signal.h
+++ b/arch/sw_64/include/uapi/asm/signal.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SIGNAL_H
 #define _UAPI_ASM_SW64_SIGNAL_H
 
diff --git a/arch/sw_64/include/uapi/asm/socket.h b/arch/sw_64/include/uapi/asm/socket.h
index abfa2108522c..a60525432c52 100644
--- a/arch/sw_64/include/uapi/asm/socket.h
+++ b/arch/sw_64/include/uapi/asm/socket.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SOCKET_H
 #define _UAPI_ASM_SW64_SOCKET_H
 
diff --git a/arch/sw_64/include/uapi/asm/sockios.h b/arch/sw_64/include/uapi/asm/sockios.h
index 1f30fb881065..88e89dcf8300 100644
--- a/arch/sw_64/include/uapi/asm/sockios.h
+++ b/arch/sw_64/include/uapi/asm/sockios.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SOCKIOS_H
 #define _UAPI_ASM_SW64_SOCKIOS_H
 
diff --git a/arch/sw_64/include/uapi/asm/stat.h b/arch/sw_64/include/uapi/asm/stat.h
index b1c1c5e3db22..d2b21128c569 100644
--- a/arch/sw_64/include/uapi/asm/stat.h
+++ b/arch/sw_64/include/uapi/asm/stat.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_STAT_H
 #define _UAPI_ASM_SW64_STAT_H
 
diff --git a/arch/sw_64/include/uapi/asm/statfs.h b/arch/sw_64/include/uapi/asm/statfs.h
index 3b8d1e3300a9..b92d719238d1 100644
--- a/arch/sw_64/include/uapi/asm/statfs.h
+++ b/arch/sw_64/include/uapi/asm/statfs.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_STATFS_H
 #define _UAPI_ASM_SW64_STATFS_H
 
diff --git a/arch/sw_64/include/uapi/asm/swab.h b/arch/sw_64/include/uapi/asm/swab.h
index a3d67645aa52..275661b346ac 100644
--- a/arch/sw_64/include/uapi/asm/swab.h
+++ b/arch/sw_64/include/uapi/asm/swab.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_SWAB_H
 #define _UAPI_ASM_SW64_SWAB_H
 
diff --git a/arch/sw_64/include/uapi/asm/sysinfo.h b/arch/sw_64/include/uapi/asm/sysinfo.h
index 9d2112f8bc4d..667405c3447c 100644
--- a/arch/sw_64/include/uapi/asm/sysinfo.h
+++ b/arch/sw_64/include/uapi/asm/sysinfo.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 /*
  * include/asm/sysinfo.h
  */
diff --git a/arch/sw_64/include/uapi/asm/termbits.h b/arch/sw_64/include/uapi/asm/termbits.h
index bcb9adb11e81..83de6ff63234 100644
--- a/arch/sw_64/include/uapi/asm/termbits.h
+++ b/arch/sw_64/include/uapi/asm/termbits.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_TERMBITS_H
 #define _UAPI_ASM_SW64_TERMBITS_H
 
diff --git a/arch/sw_64/include/uapi/asm/termios.h b/arch/sw_64/include/uapi/asm/termios.h
index d44e218b29b5..62f4b40551b2 100644
--- a/arch/sw_64/include/uapi/asm/termios.h
+++ b/arch/sw_64/include/uapi/asm/termios.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_TERMIOS_H
 #define _UAPI_ASM_SW64_TERMIOS_H
 
diff --git a/arch/sw_64/include/uapi/asm/types.h b/arch/sw_64/include/uapi/asm/types.h
index 9c605ea7bba9..750b5181c3de 100644
--- a/arch/sw_64/include/uapi/asm/types.h
+++ b/arch/sw_64/include/uapi/asm/types.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_TYPES_H
 #define _UAPI_ASM_SW64_TYPES_H
 
diff --git a/arch/sw_64/include/uapi/asm/unistd.h b/arch/sw_64/include/uapi/asm/unistd.h
index c4fdde96c640..be844b2be9d5 100644
--- a/arch/sw_64/include/uapi/asm/unistd.h
+++ b/arch/sw_64/include/uapi/asm/unistd.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 #ifndef _UAPI_ASM_SW64_UNISTD_H
 #define _UAPI_ASM_SW64_UNISTD_H
 
-- 
2.34.1

